### DISCUSSION TITLE
Tornado dependency version number

### DISCUSSION THREAD

**Discussion by dln22** [dask/distributed#7194] [Q&A]
Would the newest `distributed==2022.10.0` package work with `tornado==6.2` version? I can see there is the constraint `tornado >= 6.0.3, <6.2`, but am really hoping the package could still work with `tornado==6.2`? I am trying to resolve dependency issues in my app, and this is the last package to fix. I would very much appreciate any help. Thanks!

**Comment by ivanov** [comment]
Looks like the last bits to ensure that tornado==6.2 works are in https://github.com/dask/distributed/pull/7134
